ach/src/main/java/com/adyen/checkout/ach/ACHDirectDebitComponent.kt

```diff
@@ -32,7 +32,7 @@ import com.adyen.checkout.ui.core.internal.util.mergeViewFlows
 import kotlinx.coroutines.flow.Flow
 
 /**
- * An [PaymentComponent] that supports the 'ach' payment method.
+ * A [PaymentComponent] that supports the 'ach' payment method.
```

Comment:
I think it looks better to link the `txVariant` to `PaymentMethodTypes` using `[com.adyen.checkout.components.core.PaymentMethodTypes.ACH]`. Then people can click through the docs.